test/integration/webhooks/trainjob_test.go

```diff
@@ -63,13 +202,7 @@ var _ = ginkgo.Describe("TrainJob marker validations and defaulting", ginkgo.Ord
 	ginkgo.BeforeAll(func() {
```

Comment:
```suggestion
	var (
		trainingRuntime *trainer.TrainingRuntime
		clusterTrainingRuntime *trainer.ClusterTrainingRuntime
	)
	
	ginkgo.BeforeAll(func() {
```

Should we store the runtimes to clean up those in `BeforeAll` block?